tests/test_immutabledict.py

```diff
@@ -160,3 +162,20 @@ def test_immutabledict_duplication_blocking(self):
         immutabledict_from_unique_keys(good)
         immutabledict((x for x in good), forbid_duplicate_keys=True)
         immutabledict_from_unique_keys(x for x in good)
+
+
+def test_inverse():
```

Comment:
For consistency, these should all be methods on `class TestImmutableDict`, even though we're using pytest.